docs/general/clients/kodi.md

```diff
@@ -95,6 +95,12 @@ Native mode accesses your media files directly from the filesystem, bypassing th
 
 To use Native mode, first set up your libraries in Jellyfin with a remote path.
 
+:::caution
+
+Starting from Jellyfin 10.9 it is no longer possible to set the shared network folder.
```

Comment:
Will existing setups stop working after upgrading the server to 10.9? If so then that should be documented.

Reply:
nope, but we may remove the paths entirely in the future although nothing is decided.